src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsController.java

```diff
@@ -28,20 +32,31 @@
     @Autowired
     private NotificationService notificationService;
 
+    @Autowired
+    private HelpWithFeesBulkPrintService helpWithFeesBulkPrintService;
+
     @PostMapping(value = "/case-orchestration/notify/hwf-successful", consumes = APPLICATION_JSON_VALUE)
     @ApiOperation(value = "send e-mail for HWF Successful.")
     @ApiResponses(value = {
             @ApiResponse(code = 200, message = "HWFSuccessful e-mail sent successfully",
                     response = AboutToStartOrSubmitCallbackResponse.class)})
     public ResponseEntity<AboutToStartOrSubmitCallbackResponse> sendHwfSuccessfulConfirmationEmail(
-            @RequestBody CallbackRequest callbackRequest) {
+            @RequestBody CallbackRequest callbackRequest,
+            @RequestHeader(value = AUTHORIZATION_HEADER) String authorisationToken) {
         log.info("Received request to send email for HWF Successful for Case ID: {}", callbackRequest.getCaseDetails().getId());
         validateCaseData(callbackRequest);
         Map<String, Object> caseData = callbackRequest.getCaseDetails().getData();
         if (isConsentedApplication(caseData)) {
             if (isSolicitorAgreedToReceiveEmails(caseData, SOLICITOR_AGREE_TO_RECEIVE_EMAILS)) {
-                log.info("Sending Consented HWF Successful email notification to Solicitor");
-                notificationService.sendHWFSuccessfulConfirmationEmail(callbackRequest);
+                if (isSolicitorAgreedToReceiveEmails(caseData, "solicitorAgreeToReceiveEmails")) {
```

Comment:
use a CONST for string value

Reply:
also, this look like the same check as the if above

and the paper check below should be an `else if` rather than a nested if I believe